bin/opm

```diff
@@ -2195,6 +2198,37 @@ sub rebase_path ($$$) {
     return undef;
 }
 
+sub do_clean {
+    if (@_ == 0) {
+        err "no clean option specified.\n";
+    }
+    if ($_[0]  eq 'dist') {
+        my $dist_file = "dist.ini";
+        my $data = read_ini($dist_file);
+        my $default_sec = $data->{default};
+        my $dist_name = $default_sec->{name};
+        opendir(my $dh, './') || die "Can't opendir";
+        my @dots = readdir($dh);
+        foreach my $x (@dots) {
+            if(-d $x){
+                if ($x =~ /^\Q$dist_name\E-[.\w]*\d[.\w]*$/){
+                    shell "rm -rf $x";
+                    say 'delete:',$x;
```

Comment:
Better avoid using `say` to be consistent with the rest of the code base. Also need a space after the comma (`,`).